src/test/java/com/salesforce/dynamodbv2/testsupport/DocGeneratorRunner.java

```diff
@@ -523,11 +524,11 @@ void printToTable(List<String> columnNames, List<Object[]> data) {
             Object[][] dataArr = data.toArray(new Object[0][0]);
             TextTable tt = new TextTable(columnNamesArr, dataArr);
             ByteArrayOutputStream baos = new ByteArrayOutputStream();
-            try (PrintStream ps = new PrintStream(baos, true, "UTF-8")) {
+            try (PrintStream ps = new PrintStream(baos, true, UTF_8.name())) {
                 tt.printTable(ps, 5);
                 appendToFile(new String(baos.toByteArray()));
             } catch (UnsupportedEncodingException e) {
-                throw new RuntimeException(e);
+                e.printStackTrace();
```

Comment:
why not throw an exception here?

Reply:
addressed in https://github.com/salesforce/mt-dynamo/pull/332/commits/38481cd5105ac2f6ca44427285c1dc918a2e21ea